modules-js/react-fleet/src/form-elements/inputs/Radio.tsx

```diff
@@ -27,50 +31,75 @@ export default function Radio(props: RadioProps): JSX.Element {
         value={props.value}
         type="radio"
         className="ra-f"
+        checked={props.checked}
         onChange={props.onChange}
         onBlur={props.onBlur}
-        checked={props.checked}
       />
 
-      <span className="ra-l">{props.labelText}</span>
+      {typeof props.label === 'string' ?
+        <span className="ra-l">{props.label}</span>
+
+        :
+
+        props.label
+      }
     </label>
   );
 }
 
 
 interface RadioItem {
-  labelText: string;
+  label: string | ReactNode;
   value?: string;
+  className?: string;
   checked?: boolean;
 }
 
 interface RadioGroupProps {
   items: RadioItem[];
   name: string;
-  groupLabel?: string;
+  groupLabel: string;
+  hideLabel?: boolean;
+  checkedValue?: string;
   className?: string;
-  handleChange?(e: any): void;
+  itemsClassName?: string;
+
+  handleItemChange?(e: ChangeEvent<HTMLInputElement>): void;
+  handleItemBlur?(e: FocusEvent<HTMLInputElement>): void;
 }
 
 /**
- * Render a collection of radio buttons by providing an array of radio item information objects.
+ * Render a collection of radio buttons by providing an array of radio item
+ * information objects.
+ *
+ * Group label must be included; if it should be visually hidden, its text is
+ * added as an aria-label.
+ *
+ * If checked value is coming from a parent, pass it in as "checkedValue" prop.
  */
 export function RadioGroup(props: RadioGroupProps): JSX.Element {
+  const ariaProps = {};
+
+  props.hideLabel ? ariaProps['aria-label'] = props.groupLabel : ariaProps['aria-labelledby'] = `${props.name}-groupLabel`;
```

Comment:
This should be an `if`/`else` statement. Save the ternary operator for when you need to return a value. (Such as `const foo = flag ? 'a' : 'b'`)